src/app/users/views/summaryPartial.ejs

```diff
@@ -1,128 +1,129 @@
+        <p>
+        <form method="post" action="cancel-change-email">
+            <input type="hidden" name="_csrf" value="<%= csrfToken %>"/>
+            <button class="button-text">Cancel change email</button>
+        </form>
+        </p>
     </div>
-</div>
-<% } %>
-
-<div class="row">
-    <div class="col-8">
-        <h1 class="heading-xlarge">
-            <span class="heading-secondary"><%= user.email %></span>
-            <%= user.name %>
-        </h1>
-        <dl class="inline condensed">
-            <dt class="label">Last login:</dt>
-            <dd><%= locals.user.lastLogin ? locals.moment(locals.user.lastLogin).format('DD MMM YYYY, HH:mm:ss') : 'Never' %></dd>
-            <dt class="label">Account status:</dt>
-            <dd><%= user.status.description %></dd>
-            <dt class="label">Number of logins in last 12 months:</dt>
-            <dd><%= user.loginsInPast12Months.successful %></dd>
-        </dl>
+    <% } %>
+    <% if (locals.flash.info) { %>
+    <div class="row">
+        <div class="col-12">
+            <div class="notification notification-success">
+                <%= locals.flash.info %>
+            </div>
+        </div>
     </div>
-    <div class="col-4">
-        <% if(!locals.isInvitation) { %>
-            <aside>
-                <h2 class="heading-medium">Actions</h2>
-                <ul class="list">
-                    <li><a href="associate-organisation">Add organisation</a></li>
-                    <% if (locals.organisations.length > 0) { %>
-                        <li><a href="select-organisation">Add services</a></li>
-                    <% } %>
-                    <li><a href="edit-email">Change user's email address</a></li>
-                    <% if(locals.user.status.id === 0) { %>
-                        <li><a href="confirm-reactivation">Reactivate user</a></li>
-                    <% } else { %>
-                        <li><a href="confirm-deactivation">Deactivate user</a></li>
-                    <% } %>
-                    <li><a href="edit-profile">Edit user's details</a></li>
-                    <li><a href="web-service-sync">Send WS Sync</a></li>
-                </ul>
-            </aside>
-        <% } else if (locals.isInvitation && !locals.user.deactivated) { %>
-            <aside>
-                <h2 class="heading-medium">Actions</h2>
-                <ul class="list">
-                    <li><a href="associate-organisation">Add organisation</a></li>
-                    <% if (locals.organisations.length > 0) { %>
-                        <li><a href="select-organisation">Add services</a></li>
-                    <% } %>
-                    <li><a href="edit-email">Change user's email address</a></li>
-                    <li><a href="confirm-invitation-deactivation">Deactivate user</a></li>
-                    <li><a href="resend-invitation"> Resend invitation email</a></li>
```

Comment:
Not sure why this is being replaced when we are adding a 'text-break' class to a h1 element?

Reply:
Paragraph formatting.